workflow-manager/main.go

```diff
@@ -93,25 +181,66 @@ func main() {
 		log.Fatal(err)
 	}
 
-	var readyBatches []string
-	switch service {
+	var readyIngestionBatches []string
+	switch ingestorService {
+	case "s3":
+		readyIngestionBatches, err = getReadyIngestionBatchesS3(context.Background(), ingestorInputBucket, *ingestorIdentity)
```

Comment:
These names are starting to get pretty long. I think we can drop `ready` without loss of meaning, since the fact that we don't care about non-ready batches is an implementation detail of `getReadyIngestionBatchesS3` (or, as I propose renaming: `getIntakeBatchesS3`).